packages/browser/src/builder/BacktraceClientSetup.ts

```diff
@@ -0,0 +1,5 @@
+import { PartialCoreClientSetup } from '@backtrace-labs/sdk-core';
+import { BacktraceConfiguration } from '../BacktraceConfiguration';
+
+export interface BacktraceClientSetup<O extends BacktraceConfiguration = BacktraceConfiguration>
+    extends PartialCoreClientSetup<'sdkOptions' | 'requestHandler', O> {}
```

Comment:
we need to provide an option to override requestHandler. I think the change removes requestHandler from the browser package options. Can we bring it?